templates/chatbots/chatbot_session_view.html

```diff
@@ -44,7 +44,7 @@ <h3 class="text-base font-semibold leading-7 mt-8 pl-4">Session details</h3>
       <input type="radio" name="experiment_tabs-{{ experiment_session.id }}" role="tab" class="tab" aria-label="Messages" checked/>
       <div role="tabpanel" class="tab-content pt-3">
         <div id="messages-container" class="min-w-full"
-             hx-get="{% url 'experiments:experiment_session_messages_view' request.team.slug experiment.public_id experiment_session.external_id %}"
+             hx-get="{% url 'experiments:experiment_session_messages_view' request.team.slug experiment.public_id experiment_session.external_id %}?message_id={{ request.GET.message_id|default:"" }}"
```

Comment:
Using the `querystring` tag will preserve all query params and does the encoding for you:


```suggestion
             hx-get="{% url 'experiments:experiment_session_messages_view' request.team.slug experiment.public_id experiment_session.external_id %}{% querystring %}
```

https://docs.djangoproject.com/en/5.2/ref/templates/builtins/#querystring